src/styles/main.css

```diff
@@ -76,7 +77,7 @@ body {
         background-color: transparent;
         color: white;
         border: none;
-        font-size: 105%;
+        font-size: 108%;
```

Comment:
Looks that it doesn't fit in a/some mobile screen. Keep it as 106 with the emojis in the next line (so emojis in between the text lines). Makes sense?